### DISCUSSION TITLE
Framebuffer configuration for post processing

### DISCUSSION THREAD

**Discussion by ishax-kos** [raysan5/raylib#1815] [Q&A] [answered]
I would like to do some stylized rendering by capturing a scene's normals depth, and object origin (all in addition to color) in a screen buffer for a final post processing step. Is this possible to do in raylib?

**Comment by chriscamacho** [comment, marked as answer]
this should help with some of the techniques you will need ...
https://bedroomcoders.co.uk/cell-shader-revisited/
